src/utils/utils_common.h

```diff
@@ -77,6 +78,26 @@ static inline void *Zalloc(size_t s) {
         }                                                                      \
     } while (0)
 
+// align a pointer and a size-left
+static inline void align_ptr_size_left(void **ptr, size_t *size_left,
```

Comment:
could you rename to align_ptr_and_size()? I think the "left" word could be confusing

Reply:
Done